Assignment/solution_student/css/style.css

```diff
@@ -27,15 +27,179 @@ html {
 .clearfix:after {
   clear: both;
 }
-/******************************************
-/* BASE STYLES
-/*******************************************/
+
 body {
-  color: black;
-  font-size: 16px;
+  font-size: 13px;
+  font-weight: lighter;
   background: white;
+  padding: 50px;
+  color: #05080C;
+  line-height: 1.5;
+  max-width: 900px;
+  margin: 70px auto;
+  background: #F6F6F6;
+  border-top: 5px solid #222222;
+  font-family: 'open sans',sans-serif;
+  border-bottom: 5px solid #222222;
+  padding-top: none;
+}
+
+header {
+    height: 100px;
+    margin:0 auto;
+    padding:5px;
+    background:#003366;
+    text-align: left center;
+}
+
+.logo {
+	color: white;
+	text-indent: 5px;
+	display: inline;
+}
+
+.nav li{
+	text-align: right center;
+	display: inline-block;
+	float: right;
+	padding: 0;
+	margin: 0;
+	margin-bottom: 20px;
+	list-style: none;
+	width:auto;	
+	color:#f9e42e;
+	font-size: 15px;
+	font-weight: 300px;
```

Comment:
`font-weight` either excepts a keyword like `bold`, or **unit-less** number values in increments of 100, up to 900, so no `px` should be defined, just `300`: https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight